modules/402-ingress-nginx/monitoring/prometheus-rules/ingress-nginx.yaml

```diff
@@ -169,3 +169,15 @@
         3. If the `Number of Nodes Scheduled with Up-to-date Pods` parameter does not match
         `Current Number of Nodes Scheduled`, check the pertinent Ingress Nginx Controller's 'nodeSelector' and 'toleration' settings,
         and compare them to the relevant nodes' 'labels' and 'taints' settings
+  - alert: NginxIngressDeprecatedVersion
+    expr: d8_ingress_nginx_controller{controller_version="1.1"}
+    labels:
+      severity_level: "4"
+    annotations:
+      plk_markup_format: "markdown"
+      plk_protocol_version: "1"
+      plk_ignore_labels: "name"
+      description: |
+        Ingress version 1.1 will be disabled in Deckhouse release 1.57
```

Comment:
Usually, we provide brief description in summary, like: There is an Ingress nginx controller of a deprecated version in the cluster; and some extra information in description, like: 
There is an Ingress nginx controller of 1.1 version in the cluster. Ingress nginx verion 1.1 support will be disabled in upcoming Deckhouse releases. Consider upgrading to a newer version of Ingress nginx controller (all the pods of the affected Ingress Nginx controllers will be consequently recreated during upgrade).
Use the following command to list the affected Ingress nginx controllers: 
`kubectl  get ingressnginxcontrollers.deckhouse.io -o json | jq '.items[] | select(.spec.controllerVersion == "1.1") | .metadata.name'`